app/models/connected_app.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class ConnectedApp < ApplicationRecord
+  belongs_to :app
+  belongs_to :user
```

Comment:
Is there a reason we need to associate this to a specific user? Wouldn't this have many users through apps? I might be misunderstanding though.
```suggestion
 has_many :users, through: :apps
```

Reply:
Otherwise, all users will essentially use the same jwt for each api call. This makes it a bit more secure (I think?)

Reply:
Also this table may be used for other api tokens in the future (not just for argo)

Reply:
Ok cool, I gotcha. In that case should we rename `token` to `argo_token` or something? I could see us adding the `github_token` here eventually.

Reply:
I just made it kinda generic, but yeah I think that makes sense for sure